docs/cluster-setup-guide/deploy-cluster/index.rst

```diff
@@ -20,6 +19,31 @@ cloud, and in kubernetes environments:
 |                                          | (Slurm/PBS).                                        |
 +------------------------------------------+-----------------------------------------------------+
 
+*************
+ Basic Setup
+*************
+
+Your training environment can be a local development machine, an on-premise GPU cluster, or cloud
+resources. To set up your training environment, follow the :doc:`Basic Setup
+</cluster-setup-guide/basic>` guide.
+
+*******************************
+ Installing the Determined CLI
```

Comment:
why is this in the `deploy-cluster` section?  Presumably in any moderate-to-large installation, every user has to read about installing the determined cli, but only sysadmins have to read about installing the cluster, right?

Reply:
(Now that I built and am browsing the docs, it is also confusing there's a Basic Setup here, and another in the sidebar nav)

Reply:
removed CLI and Basic from this page